testsuite/escript/classes/fail-self-base-inheritance.err

```diff
@@ -0,0 +1 @@
+fail-self-base-inheritance.src:10:1: error: Class 'Animal' references itself as a base class through inheritance.
```

Comment:
There are actually _three_ errors: each class references itself, but we can't really check multiple errors in our testsuite currrently:
```
fail-self-base-inheritance.src:10:1: error: Class 'Animal' references itself as a base class through inheritance.
fail-self-base-inheritance.src:7:1: error: Class 'BaseClass2' references itself as a base class through inheritance.
fail-self-base-inheritance.src:3:1: error: Class 'BaseClass1' references itself as a base class through inheritance.
```